pom.xml

```diff
@@ -138,6 +232,13 @@
                 <groupId>org.alfresco</groupId>
                 <artifactId>alfresco-heartbeat-data-sender</artifactId>
                 <version>${dependency.alfresco-hb-data-sender.version}</version>
+                <exclusions>
+                    <!-- Duplicates classes from jakarta.xml.bind:jakarta.xml.bind-api -->
```

Comment:
I think the switch to jakarta can be made in heartbeat-data-sender too, since it's an alfresco lib. Maybe in a different task though :).

Reply:
I think we'll handle that at the next **jakarta** upgrade, when the the Java package namespace will change (from `javax.` to `jakarta.`). That will necessitate a new release of *alfresco-heartbeat-data-sender* (with a few updated imports).

Right now though, the *alfresco-heartbeat-data-sender* project is not quite prepared for new releases (no TravisCI build configuration).